### ISSUE TITLE
Common pieces/options for Keyboard Help content (clean up)

### ISSUE THREAD

**Issue by terracoda** [phetsims/joist#669]
@zepumph, while reviewing content for RaP's KB Shortcuts dialog, I noticed some typos/mistakes in other sims which might be in common code options.

I noticed the following:
**In Molarity**
- there are no periods in at the ends of the phrases for the combobox instructions. Unlike the visual content, the a11y content should have periods at the ends of the phrases.
- also in the combobox instructions the "Escape key" is not spelled out. It is written as "Esc key".

**In all sims with group navigation, the Basic Actions** has a typo
- There is one instance of "Arrow" that should not be capitalized.
    - "Move between items in a group with Left and Right arrow keys or Up and Down **Arrow** keys." should be:
    - "Move between items in a group with Left and Right arrow keys or Up and Down **arrow** keys."

Also, as discussed in a different issue, we will need to merge group navigation into the Basic Actions because the PhET menu is like group navigation. There is not need to separate out the arrow key navigation instructions.

Also, I was wondering in the order of things, once we merge the Basic Actions into one, if the order can be like this:
**Basic Actions**
- Move to next item or group with Tab key.
- Move to previous item or group with Shift plus Tab key.
- Move between items in a group with Left and Right arrow keys or Up and Down Arrow keys.
- Press buttons with Space key.
- Toggle checkboxes with Space key.
- Exit a dialog with Escape key.

**Comment by terracoda** [comment]
@zepumph, in the previous comment, I think "Toggle checkboxes with Space key." is an option.

**Comment by zepumph** [comment]
>Also, as discussed in a different issue, we will need to merge group navigation into the Basic Actions because the PhET menu is like group navigation. There is not need to separate out the arrow key navigation instructions.

See https://github.com/phetsims/scenery-phet/issues/630

**Comment by zepumph** [comment]
I added these fixes for Molarity/common code. I also created side issues for future work. I'm heading to https://github.com/phetsims/scenery-phet/issues/630 for general help section changes now. Closing

**Comment by zepumph** [comment]
Hey I just saw one more thing in Molarity that I'm going to add to this issue. Even though it has a checkbox in it, the general section isn't adding the row for checkboxes. I'll add that here.

**Comment by zepumph** [comment]
Alright. @terracoda, if anything here looks problematic, please reopen.
